src/ai/ai.cpp

```diff
@@ -124,7 +139,15 @@ void form_alliances(sys::state& state) {
 					else
 						return a.index() > b.index();
 				});
+				auto last = std::unique(alliance_targets.begin(), alliance_targets.end());
+				alliance_targets.erase(last, alliance_targets.end());
 				nations::make_alliance(state, n, alliance_targets[0]);
+
+				// Call our new allies into wars.... they may not accept but they may just may join!
+				for(auto wp : state.world.nation_get_war_participant(n)) {
+					if(!military::are_allied_in_war(state, n, alliance_targets[0]))
+						command::execute_call_to_arms(state, n, alliance_targets[0], wp.get_war());
```

Comment:
And don't do this; at best this is a lot of unnecessary work since most of these CTAs won't be valid